src/pages/payment-methods/_expanded-list.tsx

```diff
@@ -133,17 +134,22 @@ const ExpandList = ({ payment_data, is_fiat_onramp, locale }: PaymentProps) => {
             <Tr is_expanded={is_expanded}>
                 <Td>{payment_data.method}</Td>
                 <Td colSpan={is_fiat_onramp && parse_to_integer}>
-                    <StyleCurrencyText>{payment_data.currencies}</StyleCurrencyText>
+                    <StyleCurrencyText>
+                        {payment_data.currencies.includes('_t_') ? (
+                            <Localize translate_text={payment_data.currencies as TString} />
+                        ) : (
+                            payment_data.currencies
+                        )}
+                    </StyleCurrencyText>
                 </Td>
                 <Td>
-                    {Array.isArray(payment_data.min_max_deposit) ? (
```

Comment:
Don't we need this condition?

Reply:
By looking at the usage, min_max_deposit is always string in payment_data. That's why i remove this condition